internal/gapicgen/cmd/genbot/github.go

```diff
@@ -340,3 +319,40 @@ git push -f origin $BRANCH_NAME
 	})
 	return err
 }
+
+// MarkPRReadyForReview switches a draft pull request to a reviewable pull
+// request.
+func (gc *GithubClient) MarkPRReadyForReview(ctx context.Context, repo string, number int) error {
+	// Query to get the PR ID.
+	var q struct {
+		Repository struct {
+			PullRequest struct {
+				ID githubv4.ID
+			} `graphql:"pullRequest(number: $prNumber)"`
+		} `graphql:"repository(owner: $repositoryOwner, name: $repositoryName)"`
+	}
+	vars := map[string]interface{}{
+		"repositoryOwner": githubv4.String("googleapis"),
+		"repositoryName":  githubv4.String(repo),
+		"prNumber":        githubv4.Int(number),
+	}
+	if err := gc.cql.Query(ctx, &q, vars); err != nil {
+		return err
+	}
```

Comment:
This is fine, but as an observation, if the `PullRequest` type had a GitHull PR ID field (and `GetRegenPR` populated it), doing this query here could be skipped.

Reply:
Is the PR ID from the V3 API just the same as V4 but base64 encoded? Right now the v3 is used to get the pull request.(Although this should probably be refactored in the future so we are not using two different clients)

Reply:
Never mind decoded one from the PR mentioned in the commit body: `011:PullRequest435562529`.

I am not sure what the prefix is, maybe a version of sorts. I think I will just refactor out the v3 client in a future PR.

Reply:
Oh, it would need to be a field named "node_id" in GitHub V3 response, not "id".

As I understand GitHub docs, "id" in V3 is equivalent to [databaseId](https://developer.github.com/v4/object/pullrequest/#databaseid) in v4 (which used to be deprecated, but doesn't seem to be anymore), and "node_id" in V3 is equivalent to [id](https://developer.github.com/v4/object/pullrequest/#id) in V4. See https://developer.github.com/v4/guides/using-global-node-ids/ and https://developer.github.com/v4/scalar/id/.

I don't think it's worth making changes as part of this PR; I just wanted to share the observation.

Reply:
> (Although this should probably be refactored in the future so we are not using two different clients)

It's worth taking into account that it's been a number of years since GitHub API V4 has been released, and there are still some GitHub API features that are in-V3-but-not-yet-in-V4, and some that are in-V4-but-not-in-V3. As a result, it might be acceptable to maintain both clients even longer term, depending on your project's needs.

Reply:
Good to know, thanks for all the feedback.